repo/meta/index.json

```diff
@@ -95,6 +95,7 @@
       ],
       "versions":{
         "0.8.1":"0",
+        "0.9.0":"2",
         "0.9.0-RC3":"1"
```

Comment:
Forgive my lack of understanding. Can we get rid of the RC3 version here, or does it need to stick around since we released it once? If it can be removed, could we reuse its index number (whatever the "2" vs "1" value is)?